google/datalab/contrib/pipeline/_pipeline.py

```diff
@@ -28,48 +28,57 @@ class Pipeline(object):
 from airflow.contrib.operators.bigquery_to_bigquery import BigQueryToBigQueryOperator
 from airflow.contrib.operators.bigquery_to_gcs import BigQueryToCloudStorageOperator
 from airflow.contrib.operators.gcs_to_bq import GoogleCloudStorageToBigQueryOperator
+from google.datalab.contrib.pipeline.airflow_operators import BigQueryLoadOperator
```

Comment:
These should go under google.datalab.[contrib].bigquery.operators (to sit alongside the commands within bigquery). The class names don't need to have "BigQuery" again in their name.

Reply:
Makes sense, done. Eventually the ml operators can also reside similarly.